serde/src/de/impls.rs

```diff
@@ -70,6 +103,32 @@ impl Deserialize for () {
 
 ///////////////////////////////////////////////////////////////////////////////
 
+#[cfg(all(not(feature = "std"), not(feature = "rustc_unicode")))]
```

Comment:
The outermost function is available in `rustc_unicode`: http://manishearth.github.io/rust-internals-docs/rustc_unicode/property/fn.Pattern_White_Space.html